hack/tools/scripts/setup_cue.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+ROOT_DIR="$(dirname "${BASH_SOURCE[0]}")/../../.."
+
+wd=$(mktemp -d)
+pushd $wd
+git clone https://github.com/alexeldeib/cue
```

Comment:
https://github.com/alexeldeib/cue/commit/7d4c3a10c3ec00c7e93dc831934df7bf1ac965ba

Reply:
meh, based this on a dead upstream fork. will revise.

Reply:
fixed my fork: https://github.com/alexeldeib/cue/commit/ca3709216e49a076989a2aa422321cd991c3973e